srcpkgs/libgpg-error/template

```diff
@@ -1,28 +1,25 @@
 # Template file for 'libgpg-error'
 pkgname=libgpg-error
-version=1.37
+version=1.38
 revision=1
 build_style=gnu-configure
 short_desc="Library for error values used by GnuPG component"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later, GPL-2.0-or-later"
 homepage="https://www.gnupg.org"
-distfiles="https://www.gnupg.org/ftp/gcrypt/${pkgname}/${pkgname}-${version}.tar.bz2"
-checksum=b32d6ff72a73cf79797f7f2d039e95e9c6f92f0c1450215410840ab62aea9763
+distfiles="https://www.gnupg.org/ftp/gcrypt/${pkgname}/${pkgname}-${version}.tar.bz2
+	https://git.gnupg.org/cgi-bin/gitweb.cgi?p=libgpg-error.git;a=blob_plain;f=src/gen-lock-obj.sh;h=13858cfbf6c5f69f5f5fd64cd6fcc6a6c80eca3b>gen-lock-obj.sh"
```

Comment:
Nitpicking: variable value indentation is generally one blank, not tab stop.
Same for the checksum below.

Reply:
I have made the changes.